### PR TITLE
Recognize '@' as an operator identical to ':'

### PR DESCRIPTION
**Pull request by saxonww**
Address #785 for Apple Music users

### CODE REVIEW

File: src/DAAP2SQL.g [hunk 1]
```diff
@@ -307,6 +307,13 @@ expr	returns [ pANTLR3_STRING result, int valid ]
 
 			switch(op)
 			{
+				case '@':
```
Comment (ejurgensen) [inline src/DAAP2SQL.g]: You can probably do:
```
case '@':
case ':':
   if (neg_op) ....
```

That way duplicate code is avoided

File: src/DAAP2SQL.g [hunk 2]
```diff
@@ -273,7 +273,7 @@ expr	returns [ pANTLR3_STRING result, int valid ]
 			/* String field: escape string, check for '*' */
 			else
 			{
-				if (op != ':')
+				if (op != ':' && op != '@')
```
Comment (ejurgensen) [inline src/DAAP2SQL.g]: Can you add a comment describing why '@' is included? Paste a shortened version of the request from Apple Music, and also document the version number of Apple Music

Comment (saxonww) [reply src/DAAP2SQL.g]: Did you want the comments duplicated wherever we added '@'? I pushed an update that just adds them above line 276 here.

Comment (ejurgensen) [reply src/DAAP2SQL.g]: Fine to just have it here. Can you add the actual query? It is for future maintainers who might want to reproduce to see if the issue still exists.

**Comment by chme** [comment]
Looks good to me. I think the risk of breaking something is very low and it would be good to include it in the coming v27 release.

**Comment by ejurgensen** [comment]
Merged, thanks!

**Pull request merged**